lib/decors/decorator_base.rb

```diff
@@ -1,9 +1,11 @@
 module Decors
     class DecoratorBase
-        attr_reader :decorated_class, :decorated_method, :decorator_args, :decorator_kwargs, :decorator_block
+        attr_reader :decorated_class, :undecorated_method, :decorated_method, :decorator_args,
```

Comment:
Agree. However, "undecorated" and "decorated" terminology was already used elsewhere and I would be illogical to have `decorated_method` be the undecorated one :(